qa/rpc-tests/miningtest.py

```diff
@@ -101,12 +100,13 @@ def run_test(self):
 
         # change the coinbase
         tx = CTransaction().deserialize(c["coinbase"])
-        tx.vout[0].scriptPubKey = CScript([OP_1])
-
+        tx.vout[0].scriptPubKey = CScript(([OP_NOP] * 50) + [OP_1])  # 50 no-ops because tx must be 100 bytes or more
```

Comment:
wonder if we could use something like https://github.com/bitcoinxt/bitcoinxt/pull/487/commits/35b21eb7d5fc4de9b86abb83476852d398a27313

(I don't think we have to spend much time on this especially if we are going to remove the constraint in May 19 upgrade)